01_prepare_host.sh

```diff
@@ -201,7 +201,7 @@ function init_minikube() {
       # Loop to ignore minikube issues
       while /bin/true; do
         minikube_error=0
-        # Restart libvirtd.service as suggested here 
+        # Restart libvirtd.service as suggested here
```

Comment:
These whitespace changes are unrelated to the rest of the patch, may be worth fixing if you're doing other changes

Reply:
Yeah, let's not do it here